bin/standalone.py

```diff
@@ -360,6 +360,17 @@ def server_bind(self):
       self.socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
     BaseHTTPServer.HTTPServer.server_bind(self)
 
+  def handle_error(self, request, client_address):
+    """Handle an error gracefully. use stderr instead of stdout
+    to avoid double fault.
+    """
+    print >>sys.stderr, '-'*40
```

Comment:
Local coding convention in ViewVC doesn't employ the "print chevron" approach.  I'd rather see these lines use the sys.stderr.write() approach that's used throughout this codebase.

Reply:
Ty I'm so lost. Lol help always appreciated